Documentation/user-guides/configuring-cluster-monitoring.md

```diff
@@ -1,137 +1,858 @@
-# Configuring Cluster Monitoring
+# Cluster Monitoring Configuration Reference
 
-Parts of Cluster Monitoring are configurable. This configuration lies in a ConfigMap called `cluster-monitoring-config` in the `openshift-monitoring` namespace. The configuration file itself is defined under the `config.yaml` key within the ConfigMap's data.
+Parts of Cluster Monitoring are configurable. Depending on which part of the stack users want to configure, they should edit the following:
 
-Configuring Cluster Monitoring is optional. If the config does not exist, or is empty or malformed, then defaults will be used.
+- Configuration of OpenShift Container Platform monitoring components lies in a ConfigMap called `cluster-monitoring-config` in the `openshift-monitoring` namespace.
+- Configuration of components that monitor user-defined projects lies in a ConfigMap called `user-workload-monitoring-config` in the `openshift-user-workload-monitoring` namespace.
 
-## Configuring custom images
+The configuration file itself is always defined under the `config.yaml` key within the ConfigMap's data.
 
-In certain environments it may be required that container images are downloaded from a custom registry rather than from the canonical container image repositories on [quay.io][quay].
+Monitoring a platform such as OpenShift requires a coordination of multiple components that must work well between themselves.
+However, users should be able to customize the monitoring stack in such a way that they end up with a resilient and highly available monitoring solution.
+Despite this, to avoid users from misconfiguring the monitoring stack of their clusters not all configuration parameters are exposed.
 
-This is an example configuration with all image parameters set to a custom registry:
+Configuring Cluster Monitoring is optional. If the config does not exist or is empty or malformed, then defaults will be used.
+
+## Index
+
+- [Configuration of OpenShift Container Platform](#configuration-of-openShif-container-platform)
+  - [alertmanagerMain](#alertmanagerMain)
+  - [enableUserWorkload](#enableuserworkloadmonitoring)
+  - [k8sPrometheusAdapter](#k8sprometheusadapter)
+  - [kubeStateMetrics](#kubeStateMetrics)
+  - [prometheusK8s](#prometheusK8s)
+  - [prometheusOperator](#prometheusOperator)
+  - [openshiftStateMetrics](#openshiftStateMetrics)
+  - [thanosQuerier](#thanosQuerier)
```

Comment:
missed `telemeterClient`

Reply:
Yes, I didn't include it as it doesn't concern user configuration